### PR TITLE
update: 11.5.3

### PR DESCRIPTION
**Pull request by Danielecina**
## Pull Request Type

- [x] Documentation content changes
- [x] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [ ] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [ ] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [ ] No sensible content has been committed

### CODE REVIEW

File: docs/release-notes/v11.5.3.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+image: "img/release-note-link-preview.png"
+---
+
+_Oct 5th, 2023_
+
+## Console
+
+### Bug Fix
+
+#### Inability to create cluster from empty-state
```
Comment (fredmaggiowski) [inline docs/release-notes/v11.5.3.md:14]: let's put a warning in 11.5.0 RN that has this bug and has been fixed in v11.5.3 with a link to this page

File: docs/release-notes/v11.5.3.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
+id: v11.5.3
+title: Version v11.5.3 Release Notes
+sidebar_label: v11.5.3
+image: "img/release-note-link-preview.png"
+---
+
+_Oct 5th, 2023_
```
Comment (davidebianchi) [inline docs/release-notes/v11.5.3.md:8]:
```suggestion
_Oct 9th, 2023_
```

File: docs/release-notes/v11.5.3.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+### Bug Fix
+
+#### Inability to create cluster from empty-state
+
+Fixed a bug on the empty-state button in the Clusters section that prevented the connection of the first cluster on the respective Company.
+
+## How to update your Console
+
+For on-premise installations, please contact your Mia-Platform referent and upgrade your _Console Helm Chart_ to `v9.5.5+hotfix.1`.
```
Comment (davidebianchi) [inline docs/release-notes/v11.5.3.md:20]:
```suggestion
For on-premise installations, please contact your Mia-Platform referent and upgrade your _Console Helm Chart_ to `v9.5.5-hotfix.0`.
```

**Review by davidebianchi — CHANGES_REQUESTED** [review]

File: docs/release-notes/v11.5.3.md (hunk 1 above)
Comment (jacoposte) [reply docs/release-notes/v11.5.3.md:14]: Why? We never did that and I have never seen this practice in RNs. We should do the same also with v11.5.1 and v11.5.2, seems too much to me

**Review by davidebianchi — APPROVED** [review]

**Pull request merged**